mautic/mautic
# Integration populate blank values

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR allows the user to select if blank values in a Salesforce integration are updated on both platforms if populated in the other side regardless of the direction of the data flow configured.

#### Steps to test this PR:
1. Select update blank values in populated on either integration option
2. after a sync, if there is a blank value in Salesforce on a mapped field, and it was recently updated in Mautic, this value should be populated in Salesforce. And vice versa.
